src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.ts

```diff
@@ -13,10 +14,20 @@ export class DotTemplateBuilderComponent implements OnInit {
     permissionsUrl = '';
     historyUrl = '';
 
-    constructor() {}
+    constructor(private dotCustomEventHandlerService: DotCustomEventHandlerService) {}
```

Comment:
This is a dumb component, let's keep it that way, let's just emit the custom event out so `DotTemplateCreateEditComponent` which is the parent does the redirect to edit the template.

Also we don't need to modify `DotCustomEventHandlerService` let's keep all centralize into the templates portlet just do the redirect using the router service in `DotTemplateCreateEditComponent`.